lib/compat/wordpress-6.3/class-gutenberg-rest-global-styles-revisions-controller-6-3.php

```diff
@@ -21,7 +21,7 @@ class Gutenberg_REST_Global_Styles_Revisions_Controller_6_3 extends WP_REST_Cont
 	 * @since 6.3.0
 	 * @var string
 	 */
-	private $parent_post_type;
+	protected $parent_post_type;
```

Comment:
Why do we need this change?

Reply:
Cause we use that variable to create the schema in `Gutenberg_REST_Global_Styles_Revisions_Controller_6_4` that extends that one.

Reply:
Gotcha, yes that makes sense of course 🙂